husky_components/matrix/main.js

```diff
@@ -187,9 +189,9 @@ define(function() {
         prepareTableRow: function($tbody, rowCount) {
             var $tr = this.sandbox.dom.createElement('<tr/>'),
                 $tdHead = this.sandbox.dom.createElement('<td class="section"/>'),
-                $tdValue = this.sandbox.dom.createElement('<td class="value"/>'),
-                $tdAll = this.sandbox.dom.createElement('<td class="all"/>'),
-                allActive;
+                $tdValue = this.sandbox.dom.createElement('<td class="value"></td>'),
+                $tdAll = $tdValue,
```

Comment:
How does that make sense? I am quite sure these two elements should not be the same.